src/Auth0.OidcClient.iOS/Auth0.OidcClient.iOS.csproj

```diff
@@ -1,6 +1,6 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+<Project Sdk="Xamarin.Legacy.Sdk/0.2.0-alpha4">
```

Comment:
This is alpha but needed to ensure our SDK works on both Xamarin and newer .NET iOS. It's most likely going to stay alpha as Xamarin is going to be deprecated and investment in tooling for running it on Xamarin is limited.